modules/webgl/src/adapter/resources/webgl-command-buffer.ts

```diff
@@ -181,7 +183,9 @@ function _copyTextureToBuffer(device: WebGLDevice, options: CopyTextureToBufferO
     );
   } finally {
     device.gl.bindBuffer(GL.PIXEL_PACK_BUFFER, null);
-    device.gl.bindFramebuffer(GL.FRAMEBUFFER, null);
+    if (prevHandle !== undefined) {
```

Comment:
Hmm - When do we get undefined? When not instrumented? 

Can we add a comment explaining why we do not reset in this case.

Reply:
The assignment of `prevHandle` is in a try/catch so the code may not get executed.